front/client/components/CategoryInfoEdit.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <v-dialog
+    :value="value"
+    @input="close"
+    max-width="500px"
+  >
+    <slot slot="activator" />
+    
+    <v-card>
+      <v-card-text>
+        <v-form
+          lazy-validation
+          v-model="isValid"
+          @keydown.native.prevent.enter="submit"
+        >
+          <v-text-field
+            autofocus
+            class="mb-2"
+            label="Title"
+            :rules="itemValidationRules"
+            v-model="title"
+          />
+          <v-text-field
+            autofocus
+            class="mb-2"
+            label="Group"
+            :rules="itemValidationRules"
+            v-model="group"
+          />
+          <v-select 
+            :items="items"
+            v-model="categoryStatus"
+            label="Status"
+            class="mb-2"
+          />
+          <v-checkbox
+            v-model="prosCons"
+            label="Pros/cons enabled"
+            class="mb-2"
+          />
+          <v-checkbox 
+            v-model="ecosystem"
+            label="Ecosystem field enabled"
+            class="mb-2"
+          />
+          <v-checkbox 
+            v-model="notes"
+            label="Notes field enabled"
+            class="mb-2"
+          />
+        </v-form>
+      </v-card-text>
+      <v-card-actions>
+        <v-spacer />
+        <v-btn
+          flat
+          color="primary"
+          class="add-cat-submit"
+          :disabled="!isValid"
+          @click="updateCategoryInfo"
+        >
+          Submit
+        </v-btn>
+        <v-btn
+          flat
+          color="primary"
+          @click.native="close"
+        >
+          Cancel
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template> -->
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import _remove from 'lodash/remove'
+import _includes from 'lodash/includes'
+
+@Component
+export default class CategoryInfoEdit extends Vue {
+  @Prop(Boolean) value!: boolean
+  @Prop(String) categoryId!: string
+
+  title: string = ''
+  group: string = ''
+  categoryStatus: string = ''
+  prosCons: boolean = true
+  ecosystem: boolean = true
+  notes: boolean = true
+  sections: any[] = []
+  isValid: boolean = false
+
+  items = ['Complete', 'Work in progress', 'Stub']
+
+  itemValidationRules: Function[] = [
+    (x: string) => !!x || 'Item name can not be empty'
```

Comment:
You call both 'Title' and 'Group'. Make accurate validation rules for each input (you can even do it in markup)

Reply:
done